### PR TITLE
create API for profiles

### LINKED ISSUE #5: Profile Schema
## Description

We need to create the endpoints for creating a new profile and getting a profile.

## Spec

- [ ] POST: /v1/profiles with a _valid json object_ for a user which _does not have a profile_ is successful
- [ ] POST: /v1/profiles fails if the json is malformed
- [ ] POST: /v1/profiles fails if there is already a profile
- [ ] GET: /v1/profiles/$id produces a user's profile, if it exists.
- [ ] GET: /v1/profile produces the current user's profile.

### PR DESCRIPTION
**Pull request by Ross3102**
Creates an API for getting the currently logged-in user's profile (for now just a dummy value), a specific profile by ID, and creating a new profile.

Closes #5

### CODE REVIEW

**Review by connorlbark — CHANGES_REQUESTED** [review]
thoughts:
- separate typescript model
- jest mockups to make sure the end points work as expected

**Comment by connorlbark** [comment]
re discussion: separate profile photo endpoint? didn't specify that in the original ticket, but might make sense to add.

File: routes/profiles.route.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    .catch((err) => {
+      res.status(404).json(err);
+    });
+});
+
+// Gets the profile with the given id
+router.get('/:id', async (req: Request, res: Response) => {
+  let id = req.params.id;
+
+  ProfileModel.findOne({ _id: id })
```
Comment (nicoledanuwidjaja) [inline routes/profiles.route.ts:23]: For future ref: database logic should be handled in the profile controller in `/controllers`.
Note for discussion: How do we want to have a separate API model specifically called for different endpoints?

Comment (nicoledanuwidjaja) [reply routes/profiles.route.ts:23]: Also noting that we want to add [OpenAPI](https://swagger.io/specification/) docs in the future

**Review by nicoledanuwidjaja — COMMENTED** [review]
Comments!

**Pull request closed without merging**